.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "eslint-config-keep",
+    "plugin:react/recommended"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2017,
+    "sourceType": "module"
+  },
+  "rules": {
+    // Fixes eslint no-invalid-this error for fat arrow class methods
+    "no-invalid-this": 0
```

Comment:
I see this is related to our old components… Would it be crazy to go ahead and rewrite those? I'm not sure if that's too big a task atm.

Reply:
Yeah I think it's too big a test, took a closer look heh.

Reply:
👌 We can do it gradually :)